app/routes/dashboards/dashboard.js

```diff
@@ -2035,7 +2035,11 @@ export default Ember.Route.extend({
 
                     let parameter_value;
                     if (param.parameterName in transition.queryParams) {
-                        parameter_value = transition.queryParams[param.parameterName];
+                        if (param.parameterName == "page") {
```

Comment:
It's safer to use `===` for comparison in JS since that will compare both type and value